### PR TITLE
Splitting the RD's Office access and Server Access (also removes the Unused Apothecary access)

### PR DESCRIPTION
**Pull request by PestoVerde322**
## Why It's Good For The Game

_(NOTE: the last version broke itself https://github.com/BeeStation/BeeStation-Hornet/pull/5857, so I had to redo the pull request)._

Separate the access between the RD's office and the Research Server's access for the purpose of giving scientists dedicated server room instead of pestering the RD or building a ghetto server room in the Xenobiology's kill room or in the R&D lab (with the new access code 70).
Also it removed the unused "ACCESS_APOTHECARY", which had the same code as "ACCESS_RC_ANNOUNCE" (aka the access code 59), meaning that someone who had the access to the Apothecary could in theory also access the Request console's announcement system.

## Changelog
:cl:
add: Added a new Access code for the Research Servers, separating the server room access and the RD's office access, which on roundstart is available only to the Captain and the Research director.
tweak: Adjusted all the maps to have the right access to the server room, where it's now needed either the RD's office access or the Server room access.
del: Removed the Apothecary access, as it was used nowhere and was conflicting with the Request console's announcement system.
/:cl:

### CODE REVIEW

File: code/modules/research/server.dm [hunk 1]
```diff
@@ -30,7 +30,7 @@
 	var/temp_tolerance_damage = T0C + 200		// Most CPUS get up to 200C they start breaking.  TODO: Start doing damage to the server?
 	var/temp_penalty_coefficient = 0.5	//1 = -1 points per degree above high tolerance. 0.5 = -0.5 points per degree above high tolerance.
 	var/current_temp = -1
-	req_access = list(ACCESS_RD) //ONLY THE R&D CAN CHANGE SERVER SETTINGS.
+	req_access = listlist(ACCESS_RD_SERVER) //ONLY THE R&D, AND WHO HAVE THE ACCESS TO CAN CHANGE SERVER SETTINGS.
```
Comment (Archanial) [inline code/modules/research/server.dm]:
```suggestion
	req_access = list(ACCESS_RD_SERVER) //ONLY THE R&D, AND WHO HAVE THE ACCESS TO CAN CHANGE SERVER SETTINGS.
```

**Comment by ItsTisme** [comment]
meta station is on map freeze btw

**Comment by Archanial** [comment]
> meta station is on map freeze btw

this pr is exempt from the freeze

**Review by Archanial — APPROVED** [review]
no more weird diffs I like it

**Pull request merged**